charts/actions-runner-controller/values.yaml

```diff
@@ -15,6 +15,11 @@ image:
   dindSidecarRepositoryAndTag: "docker:dind"
   pullPolicy: IfNotPresent
 
+kube_rbac_proxy:
```

Comment:
Very nit but let me change this to `kubeRBACProxy` (lowerCamelCase) for consistency with other keys 🙏